templates/repo/scripts/component.mk

```diff
@@ -36,7 +36,10 @@ get: ssh-forward
 plan: fmt get init ssh-forward
 	$(docker_terraform) plan
 
-apply: fmt get init ssh-forward
+apply: fmt get init ssh-forward real-apply
```

Comment:
think this indirection might bring more complexity than benefit. 
running apply still requires the union of all its sub-command capabilities - you'd still need tty capabilities in order to run `apply` even if only allocated on `real-apply`.

Reply:
real-apply removed in latest. Please re-review and approve.